src/main/java/fr/moribus/imageonmap/image/MapInitEvent.java

```diff
@@ -97,11 +98,14 @@ public static void initMap(MapView map) {
 
     @EventHandler
     public void onChunkLoad(ChunkLoadEvent event) {
-        for (Entity entity : event.getChunk().getEntities()) {
-            if (entity instanceof ItemFrame) {
-                initMap(((ItemFrame) entity).getItem());
+        //Fix for paper
```

Comment:
Fix for paper where the item frame doesn't update if the chunk is loaded at the same time as the render is attached to the frame entity.